community/jmx/src/main/resources/META-INF/services/org.neo4j.kernel.KernelExtension

```diff
@@ -1 +0,0 @@
-org.neo4j.jmx.impl.JmxExtension
```

Comment:
There is no service provider for `KernelExtension` so this looks like a relic of the past. While verifying this I found 9 more leftovers that I will include in a separate PR.